docker/README.md

```diff
@@ -0,0 +1,95 @@
+# Docker container description for Tendermint applications
+
+* Overview (#Overview)
+* Tendermint (#Tendermint)
+* Basecoin (#Basecoin)
+* Ethermint (#Ethermint)
+
+## Overview
```

Comment:
Maybe it would be cool to extend this with `docker-compose.yaml` files to orchestrate this properly.